lib/pure/os.nim

```diff
@@ -338,6 +338,47 @@ proc expandFilename*(filename: string): string {.rtl, extern: "nos$1",
       result = $r
       c_free(cast[pointer](r))
 
+proc normalizePath*(path: var string) {.rtl, extern: "nos$1", tags: [].} =
```

Comment:
this should be in ospaths, no? nothing in `normalizePath` accesses file system

Reply:
oh yeah, I forgot about this. But @FedericoCeratto has waited long enough with this PR so we can fix this in a separate PR.

Reply:
of course; I much prefer faster PR turnaround than waiting forever for PR's to be so-called "perfect" (this slow turnaround really hurts D ecosystem IMO, Nim seems better in that regard)

Reply:
=> kept track of that in https://github.com/nim-lang/Nim/issues/8177 meta-issue so we don't forget